applications/luna-low-code/frontend/src/_css-variables.scss

```diff
@@ -25,6 +25,7 @@
     --kyanite-primary-90-l: 92%;
     --kyanite-navbar-height: 6.25rem;
 
+    --color-primary--500: #1D1F31;
```

Comment:
Is this some leftover? Why it is the same as `--color-default`, do we use it?

Reply:
Yes, I don't know why it has the same value, so I changed the usages to --color-default and removed this value